ckanext/msl_ckan/schemas/organizations/cohort_study.yml

```diff
@@ -10,77 +10,127 @@ fields:
   form_placeholder: eg. economy, mental health, government
 - field_name: url
-  label: Details
+  label: URL to cohort study homepage
   form_placeholder: http://example.com/dataset.json
   display_property: foaf:homepage
   display_snippet: link.html
 # name is an internal CKAN identifier;
 # if the cohort has a DOI then possibly use hash of this
+# DH: perhaps skip "name" as we already have "title", or replace title with name?
 - field_name: name
   label: Identifier
   preset: dataset_slug
   form_placeholder: eg. my-dataset
 
 ### CD² specific fields
 - field_name: cohort_study_title
-  label: Title of the cohort study
+  label: Title of the cohort study as a whole
   preset: title
   form_placeholder: eg. A descriptive title
 - field_name: cohort_study_alternate_title
-  label:  Alternate title
+  label:  Alternative title of the cohort study as a whole
 - field_name: cohort_study_principal_investigators
-  label: Principal investigators
+  label: Creators of the cohort study, e.g., principal investigators
   repeating_subfields:
   - field_name: cohort_study_pi_name
-    label: Name
+    label: Full name of the creator
     required: true
   - field_name: cohort_study_pi_affiliation
-    label: Affiliation
+    label: Affiliation of the creator
     required: true
   - field_name: cohort_study_pi_pid
-    label: Identifier
+    label: Identifier of the creator (e.g., ORCID)
   - field_name: cohort_study_pi_identifier_type # ORCID, SCOPUS, etc.
-    label: Identifier type
+    label: Type of identifier used for the creator, e.g., ORCID
+
+# DH: Contributors are also in the metadatamodel under Study / Organization
+# Copy-paste from cd2_cohort.yml
+#- field_name: contributors
+#  label: Contributors of the cohort study
+#  repeating_subfields:
+#  - field_name: contributor
+#    label:  Full name of the contributor
+#    required: true
+#  - field_name: contributor_affiliation
+#    label: Affiliation of the contributor
+#    required: true
+#  - field_name: contributor_pid
+#    label: Identifier of the contributor (e.g., ORCID)
+#  - field_name: contributor_identifier_type # ORCID, SCOPUS, etc.
+#    label: Type of identifier used for the contributor, e.g., ORCID
+
+# DH: Publisher is also in the metadatamodel under Study / Organization
+# Copy-paste from cd2_cohort.yml
+#- field_name: publisher
+#  label: Publishing organization (of the cohort data)
+#  repeating_subfields:
+#  - field_name: publisher_name
+#    label: Name of organization hosting the data
+#    required: true
+#  - field_name: publisher_identifier
+#    label: Identifier of the organization
+#  - field_name: publisher_identifier_type
+#    label: Type of organization identifier (e.g., PIC, GRID)
+
+# DH: Language is also in the metadatamodel under Study / Organization
+#- field_name: cohort_study_language
+#  label: Language of the study
+
 - field_name: cohort_study_international_identifier
   label: Cohort study international identifier # e.g. DOI
```

Comment:
Within EPOS we are currently moving to single field identifiers, e.g. 'DOI' instead of <identifier><value/><type/></identifier>. For personal identifier we will use an ORCID field and a SCOPUS field. The problem here is that we do not know at this stage what could be the most appropriate standard to use. A very important discussion, but perhaps it is good to find out the status via OpenAIRE first.